gtsfm/frontend/verifier/ransac.py

```diff
@@ -54,7 +57,7 @@ def verify(
         match_indices: np.ndarray,
         camera_intrinsics_i1: Cal3Bundler,
         camera_intrinsics_i2: Cal3Bundler,
-    ) -> Tuple[Optional[Rot3], Optional[Unit3], np.ndarray]:
+    ) -> Tuple[Optional[Rot3], Optional[Unit3], np.ndarray, float]:
```

Comment:
need to add description of the new returned variable

Reply:
thanks -- should be fixed now